packages/button/src/css/index.ts

```diff
@@ -13,18 +13,10 @@ import {
   names as themeNames
 } from '@pluralsight/ps-design-system-theme'
 import { transparentize } from '@pluralsight/ps-design-system-util'
+import { NoEmitOnErrorsPlugin } from 'webpack'
```

Comment:
Was this added by mistake?

Reply:
I guess so. I'm not sure how that got added. I'll remove it.